src/vaadin-combo-box.html

```diff
@@ -34,10 +34,7 @@
         min-width: 0;
       }
 
-      [part="clear-button"],
-      [part="toggle-button"] {
```

Comment:
1. Please fix the `stylelint` warning, as the tests are failing in CI:
```
Unexpected "       "                  selector-descendant-combinator-no-non-space
 19:28  ✖  Expected no more than 0 empty lines   selector-max-empty-lines
```

2. The code below for `::before` and `::after` pseudo elements` with code like `content: "\e901";` should be actually removed as well. It is safe to remove as it is overridden by themes, e.g. Lumo:
https://github.com/vaadin/vaadin-combo-box/blob/e48b108ec8a3eef785979af0e0bf65e0e48ce8a9/theme/lumo/vaadin-combo-box-styles.html#L14